multimodal/src/autogluon/multimodal/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -69,6 +69,20 @@ model:
       # - "random_delete(0.05)"        # less than 0.1 based on eda paper
       # - "syn_replacement(0.05)"  # less than 0.1 based on eda paper
       # - "insert_punc(0.05)"
+  ner:
+    checkpoint_name: "bert-base-cased"
+    max_text_len: 512
+    gradient_checkpointing: False
+    data_types:
+      - "text"
+    tokenizer_name: "hf_auto"
+    insert_sep: False
+    text_segment_num: 2
+    stochastic_chunk: False
+    num_classes: 1
```

Comment:
Is this `num_classes ` necessary? I don't see it being used.

Reply:
yeah, it is not necessary. I will remove it.